### DISCUSSION TITLE
Documentation on compiling?

### DISCUSSION THREAD

**Discussion by raszpl** [smplayer-dev/smplayer#916] [Q&A]
Is there a page somewhere explaining what do I need to compile smplayer under windows? Compiler and libraries required, detailed instructions if required?

**Comment by smplayer-dev** [comment]
https://raw.githubusercontent.com/smplayer-dev/smplayer/master/Install.txt

There's also this repo with scripts to help build smplayer on Windows:
https://github.com/smplayer-dev/smplayer-build
